microprofile.ls/org.eclipse.lsp4mp.ls/src/main/resources/org/eclipse/lsp4mp/snippets/jakarta-rest.json

```diff
@@ -24,7 +24,8 @@
       "type": [
         "javax.ws.rs.GET",
         "jakarta.ws.rs.GET"
-      ]
+      ],
+      "contentType": "CLASS"
```

Comment:
I would prefer writing `"contentType": "class"`.

Reply:
What do you think of using Gson to serialize/deserialize? This information is not sent between microprofile-ls and the jdt component.

Reply:
If you wish, but I think it should be better to write `"contentType": "class"`